telemetry/jetbrains/build.gradle.kts

```diff
@@ -143,3 +144,13 @@ signing {
         sign(publishing.publications["maven"])
     }
 }
+
+nexusStaging {
+    packageGroup = "software.aws"
+    serverUrl = "https://aws.oss.sonatype.org/service/local/"
```

Comment:
Do we need to care about snapshot vs release?

Reply:
No, since we don't build up multiple artifacts and release at the same time